tpu/utils/README.md

```diff
@@ -0,0 +1,24 @@
+# Utilities
+
+This directory contains helpful utilities to use [Cloud TPU](https://cloud.google.com/tpu/).
+
+
+## TPU Utilities
+
+[`tpu_utils.py`](tpu_utils.py) contains Python wrapper functions for the [Cloud TPU REST API](https://cloud.google.com/tpu/docs/reference/rest/) to manage the TPU resources.  This allows programatically creating, deleting, and monitoring TPU nodes.
+
+
+## TPU Survival Training
```

Comment:
Are the survival files a demo of using TPU utils or part of the utility library?

Reply:
They are one possible ways to use the TPU utils.  I find the directory structure not conveying this clearly to the user, but don't have a good way of organizing this to make `tpu_utils.py` importable by `tpu_survival.py`.

Reply:
Ah... So this is like a demo. 

hmmm it'd be nice to put it in a `/demo` folder or maybe name it like `tpu_utils_demo.py` and part of the README says this is an example on how to use the library?

Reply:
not so much a demo either, I think.  `tpu_utils.py` are convenient wrappers around the low level automatically generated client library introduced in this PR, and `tpu_survival.py` is (1) a good application of `tpu_utils.py` and (2) an important pattern of using preemptible Cloud TPUs, which has been asked for by users.